README.md

```diff
@@ -78,11 +94,20 @@ if you want to download that file and follow along.
+*	On the selection of *k*-mer length: In our experience, the most robust biological trends have been relatively insensitive to the length of *k*-mer used in SEEKR. Still, when deciding on a length of k to use for comparisons, we recommend using a *k*-mer length for which 4^k is similar to the length of the average feature or key feature that is being compared. The reason for this is that as the length of k increases, so does the number of zero values for *k*-mer counts in a given sequence. For example, there are 16384 possible 7-mers. If users were interested in finding lncRNAs that are similar to lncRNA-X, which is 500 nucleotides long, a *k*-mer length of 7 would not be ideal, because the vector of 7-mer counts that corresponds to lncRNA-X would be dominated by zero values. In this example, unless users had a specific rationale for searching 7-mers, a *k*-mer length of 4 (256 possible *k*-mers) or 5 (1024 possible *k*-mers) would provide the basis for a stronger comparison.
+
 
 ### Help
 
-For full documentation of the parameters and flags, you can run any of the commands without any arguments.
-For example:
+Please also see [the pre-print](https://github.com/CalabreseLab/seekr/blob/logchanges/methods_mol_bio_seekr-v20.pdf) to a methods paper we wrote last year. This paper was originally scheduled to appear in Methods in Molecular Biology in 2020 but its publication date may be delayed. 
+
+If you have questions about how you can use seekr in your own research, please send an email to jmcalabr@med.unc.edu
+
+For full documentation, type...
```

Comment:
did this just trail off?

Reply:
no, mauro just used an ellipsis. I changed it to a colon